extensions/schema-compare/src/schemaCompareMainWindow.ts

```diff
@@ -671,7 +671,7 @@ export class SchemaCompareMainWindow {
 					.withAdditionalProperties({
 						'operationId': this.operationId
 					}).send();
-				vscode.window.showErrorMessage(loc.cancelErrorMessage(result.errorMessage));
+				this.apiWrapper.showErrorMessage(loc.cancelErrorMessage(result.errorMessage));
```

Comment:
I think there was a plan to move away from ApiWrapper. I know that this isn't part of the PR, but should we write new tests without the use of apiWrapper?

Reply:
Yes please 😄

Reply:
sure I can do that.